### ISSUE TITLE
ESRI .prj not being interpreted correctly "AEA [SHG]"

### ISSUE THREAD

**Issue by mogikanin** [DotSpatial/DotSpatial#505]
**This issue was imported from [CodePlex](http://dotspatial.codeplex.com/workitem/22934)**

**[Woodman5k](http://www.codeplex.com/site/users/view/Woodman5k)** wrote 2012-06-22 at 23:04
Hi, I think this will be an easy fix that will hopefully be addressed soon. The following ESRI projection is not being converted correctly and is therefore incorrectly reprojecting coordinates. I narrowed the issue down to the central parallel (lat_0)
 not being included. When converting to a proj4 string, the lat_0 is not being defined whereas according to the esri projection information it is 23. By feeding dotspatial a proj4 string that includes lat_0=23.0 it was reprojecting correctly.

 

PROJCS["Albers Equal-Area Conic [SHG]",GEOGCS["GCS_North_American_1983",DATUM["D_North_American_1983",SPHEROID["GRS_1980",6378137.0,298.257222101]],PRIMEM["Greenwich",0.0],UNIT["Degree",0.0174532925199433]],PROJECTION["Albers"],PARAMETER["False_Easting",0.0],PARAMETER["False_Northing",0.0],PARAMETER["Central_Meridian",-96.0],PARAMETER["Standard_Parallel_1",29.5],PARAMETER["Standard_Parallel_2",45.5],PARAMETER["Central_Parallel",23.0],UNIT["Meter",1.0]]

**[Woodman5k](http://www.codeplex.com/site/users/view/Woodman5k)** wrote 2012-06-22 at 23:07
Simply put, the central parallel (lat_0 in proj4 language) is being ignored when reading into dotspatial. Therefore reprojecting coordinates yields incorrect results.

**[mudnug](http://www.codeplex.com/site/users/view/mudnug)** wrote 2012-07-03 at 21:50
I noted a similar issue in another project: <a href="http://trac.osgeo.org/gdal/ticket/3191" rel="nofollow">
http://trac.osgeo.org/gdal/ticket/3191</a> 

and a number of parameters that might be considered equivalent: <a href="http://support.esri.com/en/knowledgebase/techarticles/detail/39992" rel="nofollow">
http://support.esri.com/en/knowledgebase/techarticles/detail/39992</a> 

Please verify 67860 fixes the issue.
